docs/api-docs/api-docs.yml

```diff
@@ -0,0 +1,536 @@
+openapi: 3.0.0
+info:
+  description: Context aware, pluggable and customizable PII anonymization service for text and images.
+  version: "2.0"
+  title: Presidio
+  contact:
+    name: Presidio support
+    email: presidio@microsoft.com
+    url: https://github.com/microsoft/presidio
+  x-logo:
+    url: "https://upload.wikimedia.org/wikipedia/commons/4/44/Microsoft_logo.svg"
+  license:
+    name: MIT
+    url: 'https://github.com/microsoft/presidio/blob/master/LICENSE'
+externalDocs:
+  description: Presidio documentation.
+  url: 'https://github.com/microsoft/presidio/' #TODO: Link to GH pages mkdocs
+tags:
+  - name: Analyzer
+    description: Detecting PII entities in text
+  - name: Anonymizer
+    description: Anonymizing detected PII text entities with desired values.
+
+paths:
+
+  /analyze:
+    post:
+      servers:
+        - url: https://presidio-analyzer-prod.azurewebsites.net
+      tags:
+        - Analyzer
+      summary: "Analyze Text"
```

Comment:
```suggestion
      summary: "Detect PII in text"
```

Reply:
It's better to have the summary described as closest to the endpoint, as it's service as a navigator when collapsed:
![image](https://user-images.githubusercontent.com/15013757/108620534-6e723800-7435-11eb-8b8f-6024a8229d96.png)
![image](https://user-images.githubusercontent.com/15013757/108620542-792ccd00-7435-11eb-91b1-c6746b0f8a53.png)
WDYT?

Reply:
Sure, so let's be more explicit on the description.